### PR TITLE
chore(deps): update fxa-geodb

### PR DESCRIPTION
**Pull request by philbooth**
Updates `fxa-geodb` to `1.0.0` and removes the associated promise-handling code.

@mozilla/fxa-devs r?

### CODE REVIEW

File: lib/geodb.js [hunk 1]
```diff
@@ -4,61 +4,54 @@
...
+      } else if (accuracy > ACCURACY_MIN_KM && accuracy <= ACCURACY_MAX_KM) {
+        confidence = 'uncertain'
+      } else if (accuracy <= ACCURACY_MIN_KM) {
+        confidence = 'confident'
+      } else {
+        confidence = 'no_accuracy_data'
+      }
 
-        log.info({op: 'geodb.accuracy', 'accuracy': accuracy})
-        log.info({op: 'geodb.accuracy_confidence', 'accuracy_confidence': logEventPrefix + logEvent})
```
Comment (vbudhram) [inline lib/geodb.js]: We could probably update [graphs](https://kibana-fxa-us-west-2.prod.mozaws.net/#/dashboard/elasticsearch/FxA%20GeoDB%20Stats) to reflect new stats after shipping this.

File: lib/routes/account.js [hunk 2]
```diff
@@ -277,55 +277,52 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
 
         function sendVerifyCode () {
           if (! account.emailVerified) {
-            return request.app.geo
```
Comment (vbudhram) [inline lib/routes/account.js]: Much cleaner!

File: lib/geodb.js (hunk 1 above)
Comment (philbooth) [reply lib/geodb.js]: Oh, didn't realise there were graphs depending on this. Do you reckon I should put it back how it was? I thought one log line was better than two, but probably not if it breaks dashboard continuity.

Comment (philbooth) [reply lib/geodb.js]: Fwiw I reverted this.

**Pull request merged**